mdtraj/core/topology.py

```diff
@@ -1111,6 +1111,30 @@ def find_molecules(self):
             molecules[atom_molecule[atom.index]].add(atom)
         return molecules
 
+    def guess_anchor_molecules(self):
+        """Guess anchor molecules for imaging
```

Comment:
If I didn't already know, I'd be confused about what that description was talking about.  Perhaps add a "see also" referencing `Trajectory.image_molecules()`?